post/index.go

```diff
@@ -255,6 +255,53 @@ func (repository *Repository) SetPostImageInPosts(posts []Post) []Post {
 	return posts
 }
 
+func (repository *Repository) SetPostImageInRecommendedPosts(recommendedPosts []RecommendedPost) []RecommendedPost {
+
+	postIds := []string{}
+	for _, post := range recommendedPosts {
+		postIds = append(postIds, strconv.Itoa(post.Id))
+	}
+
+	newPostIds := strings.Join(postIds, ", ")
+
+	images := []Image{}
+	postImageQuery := fmt.Sprintf(`SELECT f.id, frm.related_id as related_id, f.name, f.alternative_text, f.caption, f.width, f.height, f.formats as format, f.hash, f.ext, f.mime, 
```

Comment:
Do we need all of this for recommended posts? Can't we get all of this in a single query with recommended pots?